recipe/magento2.php

```diff
@@ -454,6 +455,17 @@ function magentoDeployAssetsSplit(string $area)
     //set prefix to `alias_releasename_`
     $prefixUpdate = get('alias') . '_' . get('release_name') . '_';
 
+    //check for preload keys and update
+    if (isset($envConfigArray['cache']['frontend']['default']['backend_options']['preload_keys'])) {
+        $oldPrefix = $envConfigArray['cache']['frontend']['default']['id_prefix'];
+        $preloadKeys = $envConfigArray['cache']['frontend']['default']['backend_options']['preload_keys'];
+        $newPreloadKeys = [];
+        foreach ($preloadKeys as $preloadKey) {
+            $newPreloadKeys[] = str_replace($oldPrefix, $prefixUpdate, $preloadKey);
```

Comment:
to not accidentally replace parts of the cache keys if they match the old prefix, I would consider only replacing the prefix at the beginning of the string. Unlikely, I know, but using `substr_replace` or a regular expression seems more appropiate to me

Reply:
Thanks @schmengler 
That's now updated